src/dotenv/cli.py

```diff
@@ -45,8 +51,15 @@ def list(ctx: click.Context) -> None:
             ctx=ctx
         )
     dotenv_as_dict = dotenv_values(file)
-    for k, v in dotenv_as_dict.items():
-        click.echo('%s=%s' % (k, v))
+    if format == 'json':
+        print(json.dumps(dotenv_as_dict, indent=2, sort_keys=True))
+    else:
+        prefix = 'export ' if format == 'export' else ''
```

Comment:
Curious whats the difference between "export" and "shell". I under the use case, I'm if there any official references for the name used.

Reply:
With export, the variables will be visible to all child processes spawned by the shell script. This may not be desirable, as when there are secrets or configuration values that are intended for a specific purpose.

Without export, the variables will be private to the executing shell script, unless they are explicitly exported one by one.

Reply:
happy to change the names if you like, but I do see a use case for the distinction between local and exported variables.

Reply:
Alternatively, I could just add a --prefix option that adds an arbitrary string to the start of each line (ignored in the json case)...

Reply:
Understood and thanks for the explanation. 

I also did some reading [here](https://www.baeldung.com/linux/bash-variables-export). The `export` command has two options to show all exported variables in a shell environment

1. with exports  `exports -p` 
2. and without it. Just `exports`

I am happy with the current naming scheme.